components/hab/src/command/studio/enter.rs

```diff
@@ -13,12 +13,39 @@ use crate::{config,
 
 pub const ARTIFACT_PATH_ENVVAR: &str = "ARTIFACT_PATH";
 
+const AUTH_TOKEN_ENVVAR: &str = "HAB_AUTH_TOKEN";
+const BLDR_URL_ENVVAR: &str = "HAB_BLDR_URL";
```

Comment:
same question, defined in [`components/core/src/url.rs`](https://github.com/habitat-sh/habitat/blob/master/components/core/src/url.rs#L4), how do I load that pub const instead of defining it here.

Reply:
Again the compiler gives good advice:
```
error[E0425]: cannot find value `BLDR_URL_ENVVAR` in this scope
  --> components/hab/src/command/studio/enter.rs:41:29
   |
41 |     set_env_var_from_config(BLDR_URL_ENVVAR, config.bldr_url);
   |                             ^^^^^^^^^^^^^^^ not found in this scope
help: possible candidates are found in other modules, you can import them into scope
   |
1  | use crate::BLDR_URL_ENVVAR;
   |
1  | use habitat_core::url::BLDR_URL_ENVVAR;
```
And it even shows us that we've got duplicates of `BLDR_URL_ENVVAR`; one in [core/src/url.rs](https://github.com/habitat-sh/habitat/blob/master/components/core/src/url.rs#L4) and one in [hab/src/lib.rs](https://github.com/habitat-sh/habitat/blob/master/components/hab/src/lib.rs#L41). Either `use` statement will work, so take your pick. We should probably remove the duplication and move all these `*_ENVVAR` constants to one place, but it's probably best to leave that for a follow-up PR (which you're welcome to tackle if interested).

Reply:
Yep, fair enough.  This may have been where my confusion came from.

I used everything I could from `crate::*`